packages/server/src/workers/subscription.ts

```diff
@@ -147,12 +198,30 @@ export async function addSubscriptionJobs(resource: Resource, context: Backgroun
     // Never send subscriptions for audit events
     return;
   }
+
+  const systemRepo = getSystemRepo();
+  let project: Project | undefined;
+  try {
+    const projectId = resource.meta?.project;
+    if (projectId) {
+      project = await systemRepo.readResource<Project>('Project', projectId);
+    }
+  } catch (_err: unknown) {
+    project = undefined;
```

Comment:
I'm wondering if we should actually swallow this error, or if we should actually abort if project is gone?  Is there a valid case where `projectId` is an ID, but `readResource` fails? 🤔

Reply:
Alright, we're logging that now 👍 

Should probably never hit that log... probably...